server/update-user.js

```diff
@@ -55,7 +55,7 @@ function getChannelIds(teamChannels, channelNames) {
 }
 
 export default function (connectSlack, { client: hull, ship }, messages = []) {
-  _.map(messages, (message = {}) => {
```

Comment:
@mkawalek shouldn't we do `Promise.all` here and in nested functions? To make sure we are not loosing the promise chain.

Reply:
Good idea, it would be cleaner